### DISCUSSION TITLE
Understanding the "Due Today" $ amount - no pricing breakdown - very confusing

### DISCUSSION THREAD

**Discussion by cliffordfajardo** [community/community#8799] [Other Feature Feedback, Questions, & Ideas]
Hi Github team,
I am trying to better understand the pricing breakdown for sponsering individuals.
I find it very confusing that there is no price breakdown for the bill charged.

While going through the process of trying to contribute $50 / month, I was this as my "Due Today" bill:

![CleanShot 2021-12-13 at 12 11 59](https://user-images.githubusercontent.com/6743796/145881551-39d13d7a-26c6-42ff-9d54-9369006875bd.png)

According to Github's  [About billing for GitHub Sponsors](https://docs.github.com/en/billing/managing-billing-for-github-sponsors/about-billing-for-github-sponsors) page, it says:

> GitHub Sponsors does not charge any fees for sponsorships from user accounts, so 100% of these sponsorships go to the sponsored developer or organization. The 10% fee for sponsorships from organizations is waived during the beta.

I am attempting to sponser from my personal account (per the image above). Today is December 13, 2021. 
1. Where is this $279 coming from?
- $279 seems like an error?  That is nearly 50% of the year sponsorship ($600 = $50/month * 12)
- If I paid that fee, and then the $600 in June, I would be paying $879 for 6 months of sponsership, which doesn't seem correct?

2. Are github sponserships bills not prorated (take into account the day & month of the start of the sponsorship)?

Thank you 🙏

**Comment by sinoth** [comment]
I'm encountering this today. I wanted to sponsor a dev for $5/month but the "due today" charge is $60. Why?

**Reply by tomkerkhove** [reply]
💯

**Reply by sinoth** [reply]
> I'm encountering this today. I wanted to sponsor a dev for $5/month but the "due today" charge is $60. Why?

Just figured this out for my situation at least. Click your profile picture -> Settings -> Billing & plans -> Switch to monthly billing.

I'm on GitHub free so I didn't think to check my personal billing interval as I've never set it.

GitHub team -- this is a source of confusion and friction between users and sponsoring. Please add some help text or a quick link to 'Billing & plans' under the 'Due today' when sponsoring. This is an extra step that services like Patreon don't have.

**Reply by cliffordfajardo** [reply]
Ahh I see. The following:
> Just figured this out for my situation at least. Click your profile picture -> Settings -> Billing & plans -> Switch to monthly billing.

The aforementioned is still quite confusing, when I sponsor someone, from a user experience standpoint, I shouldn't need to go through so many hoops like switch from monthly to yearly. It should be seemless. In fact, I still haven't sponsored more people until I understand how this all works.
- if I have a yearly github subscription plan, that should be independent of my individual sponsorships. 

I'm sure there are tons of people who stumbled upon this issue (like @tomkerkhove , @sinoth  and I) who:
-  haven't reported the issue (friction point...time to bounce & get on with my life) 🚀 
- or don't know that Github has a [discussions section](https://github.com/github/feedback/discussions) to give feedback for the majority of the product/features

Again thanks Github team that you even have a discussions forum 💙🙏  @MylesBorins

**Reply by mackuba** [reply]
Oh man… I wouldn't have figured this out myself… thanks!

GitHub, please fix this… just one small line with info below the "due" amount would be enough. @jlord

**Comment by jlord** [comment]
Hi! We're working on improving this and it will be updated to be more clear 🔜 I'll post here again when it's shipped!

**Comment by jlord** [comment]
I'm late in my posting this update but we've shipped changes to better explain the prorated payments 🎉

**Comment by Abusayid693** [comment]
I'm experiencing similar issue, wanted to sponsor a dev at $4/month, but my billing page shows **Price after the change to monthly billing: $48**.
